helper.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs')
+const chalk = require('chalk')
+
+function getNotes() {
+    const data = fs.readFileSync('notes.json')
+    const content = data.toString()
+
+    try {
+       const jsonData = JSON.parse(content)
+       return jsonData
+    } catch (error) {
+       return []
+    }
+}
+
+function saveNotes(data) {  
+    const dataFile = JSON.stringify(data)
+    fs.writeFileSync('notes.json', dataFile)  
+}
+
+function addNotes(title, body) {
+    const notes = getNotes() 
+
+    const duplicates = notes.find(elem => {
+        if (elem.title === title) {
+            return true
+        }
+    })
```

Comment:
please simplify `find` method callback - you can make it in 1 line. 
[stackoverflow](https://stackoverflow.com/a/12462414) see first example